src/main/java/com/esri/samples/editing/update_geometries/UpdateGeometriesSample.java

```diff
@@ -108,8 +109,11 @@ public void start(Stage stage) throws Exception {
                 featureLayer.clearSelection();
                 featureLayer.selectFeature(selected);
               } else {
-                // move selected features
-                moveSelected(mapPoint);
+                FeatureQueryResult result = featureLayer.getSelectedFeaturesAsync().get();
```

Comment:
I don't think we should use a blocking `get` here. Also, we shouldn't keep track of a `selected` feature if we're just going to check for it every time.